apps/dotcom/src/components/DocumentName/DocumentName.tsx

```diff
@@ -239,6 +239,14 @@ const DocumentNameEditor = track(function DocumentNameEditor({
 		}
 	}, [documentSettings.name, editor, state.isEditing, state.name, setState])
 
+	useEffect(() => {
+		if (documentSettings.name) {
+			document.title = `tldraw - ${documentSettings.name}`
```

Comment:
super nit: use `—` instead of `-` :). (or, you could use `·`, either way)

finally, we should swap the order, it should be `${documentSettings.name} — tldraw` - that's typically the convention